website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -225,7 +225,7 @@ import (
 	...
 )
 
-func (r *MemcachedReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
+func (r *MemcachedReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
```

Comment:
We cannot do these changes before the v3/go plugin be stable and the default one.
Kubebuilder has not an issue in the 3.0.0 milestone to make the v3 alpha the default one. Should we raise that there?